packages/e/erc725js.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "erc725.js",
+  "filename": "erc725.min.js",
```

```suggestion
  "filename": "index.min.js",
```
It looks like the entrypoint file for the library is `index.js` (or `index.min.js` that we'll generate), not `erc725.min.js`